circle.yml

```diff
@@ -477,7 +477,9 @@ jobs:
       - install-dependencies: { mason: false, ccache: false }
       - run:
           name: Initialize submodules
-          command: git submodule update --init
+          command: |
+            git submodule update --init
+            git submodule foreach git submodule update --init
```

Comment:
git submodule update --init --recursive ?

Reply:
Unfortunately, there is no way to describe the depth when using `--recursive`. This means that when using that parameter, we end up fetching unnecessary sub-dependencies e.g. some dependencies of ours have `.mason` as subdependency, etc.

Reply:
With the latest change on `mapbox-base` I don't think this is needed. It will get initialized at configure time.

Reply:
No - neither `nitpick` nor `android-debug-arm-v7-buck` uses CMake scripts, thus the need for the manual git submodule initialization.